### PR TITLE
add a parser for cosmors files, add NMR parsing to gaussian

### PR DESCRIPTION
**Pull request by JacksonBurns**
Currently getting the volume, area, and energy

### CODE REVIEW

**Comment by JacksonBurns** [comment]
@oscarwumit please review and let me know if it needs any changes. We can merge this into the initial release branch and then write the database part of the code afterward.

File: README.md [hunk 1]
```diff
@@ -82,7 +82,13 @@ $3$ E0 for wavefunction methods <br>
 | Final Single Point Energy | `energy` | float | 1/job |
 | Input xyz coords | `input_coordinates` | list[list[float]] | 1/job |
 
-$1$ ignores milliseconds <br>
+#### Orca
```
Comment (oscarwumit) [inline README.md]: should this be Cosmo?

File: README.md [hunk 2]
```diff
@@ -82,7 +82,13 @@ $3$ E0 for wavefunction methods <br>
...
+
+| Quantity | Key | Type | Frequency |
+| -------- | --- | ---- | --------- |
+| Area | `area` | float | 1/job |
+| Volume | `volume` | float | 1/job |
+| Energy | `energy` | float | 1/job |
```
Comment (oscarwumit) [inline README.md:92]: Is this energy the solvation energy at 298K (Gsolv298) or something else?

Btw, I think we need to parse out the Gsolv at three temperatures as well as the "Total energy + OC corr. [a.u.]". Are these going to be added later?

**Comment by oscarwumit** [comment]
@JacksonBurns Thanks a lot. Please take a look at some minor comments.

**Comment by JacksonBurns** [comment]
@oscarwumit I don't see any comments

**Comment by oscarwumit** [comment]
That is weird. I can see them on my end. Can you take a look at the following screenshot? Thank you.

Screenshot 2024-08-26 at 15 05 47

File: README.md (hunk 2 above)
Comment (JacksonBurns) [reply README.md:92]: I'm not sure what type of energy this is, it's whatever is shown in the `Total energy + OC corr. [a.u.]` line

is there a Gsolv listed in the .cosmo file?

**Comment by JacksonBurns** [comment]
@shihchengli can you review and let me know how the NMR parsing looks?

File: fastlogfileparser/gaussian/utils/postprocessing.py [hunk 3]
```diff
@@ -38,6 +38,11 @@ def _mulliken(in_list):
     return out
 
 
+def _nmr(in_list):
+    print([[int(i[0])] + _str_list_to_floats(i[1:]) for i in in_list])
```
Comment (shihchengli) [inline fastlogfileparser/gaussian/utils/postprocessing.py]:
```suggestion
```

**Review by shihchengli — COMMENTED** [review]
Thank you @JacksonBurns! It looks good to me.

File: README.md (hunk 2 above)
Comment (JacksonBurns) [reply README.md:92]: ping @oscarwumit could you check the above?

Comment (shihchengli) [reply README.md:92]: The parser Jackson wrote for now is to parse out the area, energy, and volume from .cosmo file. Gsolv at different temperatures are saved in .tab or .out files. We can consider adding this in the other PRs.

**Review by shihchengli — APPROVED** [review]
LGTM

**Pull request merged**